packages/widgets/vite.config.ts

```diff
@@ -25,6 +25,7 @@ export default defineConfig(({ mode }) => {
 					emitCss: false,
 					compilerOptions: {
 						hydratable: true,
+						// @ts-expect-error to check cc @krampstudio
```

Comment:
cc @krampstudio , apparently the correct type is `Omit<ViteOptions, "generate">`, for now bypassed the error (not sure the error is relevant or not)

Reply:
indeed the type in vite-plugin-svelte omits it, and the [doc](https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/config.md#compileroptions) says it's non-configurable.
But we use it in [lib mode](https://vitejs.dev/guide/build#library-mode) and the type might not be accurate since the options are passed down to the svelte compiler